subsys/bootloader/fprotect/fprotect_bprot.c

```diff
@@ -22,17 +24,17 @@
  /* The number of CONFIG registers present in the chip. */
 #define BPROT_CONFIGS_NUM ceiling_fraction(BPROT_REGIONS_NUM, BITS_PER_LONG)
 #if defined(CONFIG_SB_BPROT_IN_DEBUG)
-#define ENABLE_BPROT_IN_DEBUG true
+#define ENABLE_IN_DEBUG true
```

Comment:
I'm curious, does

    #define ENABLE_IN_DEBUG defined(CONFIG_SB_BPROT_IN_DEBUG)

work? :)

Reply:
Yes, I tested it and it worked for me at least.

Reply:
Ok, no big deal.

Reply:
seems you cant do `defined` inside `#define`